vidarrbuild.json

```diff
@@ -0,0 +1,6 @@
+{
+    "names": [
+        "insertSizeMetrics_call_ready"
```

Comment:
shouldn't we have insertSizeMetrics_by_tumor_group added here?

Reply:
Thanks Peter, added it.